docs/tac.md

```diff
@@ -0,0 +1,60 @@
+## Setup
+
+Clone the repo to include submodules.
+
+``` bash
+git clone git@github.com:fetchai/agents-tac.git --recursive && cd agents-tac
+```
+
+
+Create and launch a virtual environment.
+
+``` bash
+pipenv --python 3.7 && pipenv shell
+```
+
+
+Install the package.
+``` bash
+python setup.py install
+```
+
+Error at this point.
```

Comment:
I am not sure we want this in here. The error should be resolved :)

Reply:
Indeed it is :)